### PR TITLE
Updates to expose swerve odometry content so that higher level business logic can move the robot to defined positions on the field

### PR DESCRIPTION
**Pull request by mmcadamsms**

### CODE REVIEW

File: smokey/src/main/java/frc/robot/subsystems/NavxSubsystem.java
Comment (josieammer) [inline smokey/src/main/java/frc/robot/subsystems/NavxSubsystem.java:51]: I believe line 51 is the one that always gets called and line 55 does not.  My suspicion is that line 55 (returning the inverted yaw) is inconsistent with line 51.  We should print both out and see if they are tracking in opposite directions and decide which one is correct.  I don't think this should block this PR because it's been in the code since Veer.  Can you add a TODO to in the code to check this out?

**Review by josieammer — APPROVED** [review]
Looks good to me.  Two minor changes requested.  Approving so you don't have to come back to me for the final merge.

File: smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:13]: agree

File: smokey/src/main/java/frc/robot/subsystems/NavxSubsystem.java
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/subsystems/NavxSubsystem.java:51]: Ok ... will add todo

Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/subsystems/NavxSubsystem.java:60]: done

File: smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: done

**Pull request merged**
